### PR TITLE
Advising Backend

### PR DESCRIPTION
**Pull request by rbiz4**

### CODE REVIEW

File: advising/views.py [hunk 1]
```diff
@@ -1,14 +1,98 @@
...
+                                         content=request.data['content'],
+                                         timestamp=request.data['timestamp'],
+                                         transcript=transcript)
+        comment.save()
+
+        return Response(status=status.HTTP_201_CREATED)
+
 
+class InvitedTimetableView(ValidateSubdomainMixin, RedirectToSignupMixin, APIView):
+    """ Handles the accessing of invited user timetables. """
```
Comment (miaboloix) [inline advising/views.py:98]: no class body?

File: advising/views.py [hunk 2]
```diff
@@ -1,14 +1,98 @@
...
+    def get(self, request):
+        """
+        Returns all invited transcripts for the user making the request:
+            invited_transcripts: Array of transcripts the user has been added to.
+        """
+        student = Student.objects.get(user=request.user)
+        return Response(
+            {'invited_transcripts': TranscriptSerializer(
+                student.invited_transcripts, many=True).data},
+            status=status.HTTP_200_OK)
```
Comment (miaboloix) [inline advising/views.py:48]: this doesn't grab any transcripts the user owns right? ONLY invited, right?

**Review by miaboloix — CHANGES_REQUESTED** [review]

**Comment by JJamesWWang** [comment]
Sorry I'm kind of out of the loop for this - what's the purpose of these views? Doesn't ForumView provide the same functionality of these new views?

**Comment by miaboloix** [comment]
can we confirm whether this functionality is new or not?

**Pull request closed without merging**